src/annoylib.h

```diff
@@ -48,6 +48,7 @@ typedef signed __int32    int32_t;
 #include <algorithm>
 #include <queue>
 #include <limits>
+#include <utility>
```

Comment:
what's this needed for?

Reply:
hmm, sorry, from another experiment, I'll remove this